api/src/modules/scenario-interventions/dto/create.scenario-intervention.dto.ts

```diff
@@ -54,9 +56,74 @@ export class CreateScenarioInterventionDto {
   @ApiPropertyOptional()
   percentage?: number;
 
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  materialsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  businessUnitsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  suppliersIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  adminRegionsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  sourcingLocationsIds: string[];
+
   @IsString()
   @IsOptional()
   @IsJSON()
   @ApiPropertyOptional()
   newIndicatorCoefficients?: JSON;
+
+  @IsUUID()
+  @IsOptional()
+  @ApiPropertyOptional()
+  newSupplierT1Id?: string;
+
+  @IsUUID()
+  @IsOptional()
+  @ApiPropertyOptional()
+  newProducerId?: string;
+
+  @IsUUID()
```

Comment:
I think new location type shouldn't be a UUID.

I think you can use the `LOCATION_TYPES` enum from `sourcing-location.entity.ts`